hewr/R/score_hubverse.R

```diff
@@ -87,7 +88,7 @@ score_hubverse <- function(forecast,
     scoringutils::score(
       metrics = c(
         scoringutils::get_metrics(to_score),
-        list(interval_coverage_95 = interval_coverage_95)
+        interval_coverage_95 = interval_coverage_95
```

Comment:
Did this fail before?

Reply:
No, it just struck me that since `get_metrics` returns a list this would be less verbose